_opensearch/index-templates.md

```diff
@@ -330,15 +324,14 @@ GET logs-2020-01-01
 }
 ```
 
-
 ## Index template options
 
 You can specify the following template options:
 
-Option | Type | Description | Required
-:--- | :--- | :--- | :---
-`template` | `Object` |  Specify index settings, mappings, and aliases. | No
-`priority` | `Integer` | The priority of the index template.  | No
-`composed_of` | `String array` |  The names of component templates applied on a new index together with the current template.  | No
-`version` | `Integer` | Specify a version number to simplify template management. Default is `null`. | No
-`_meta ` | `Object` | Specify meta information about the template. | No
+| Option        | Type           | Description                                                                                 | Required |
+| :------------ | :------------- | :------------------------------------------------------------------------------------------ | :------- |
+| `template`    | `Object`       | Specify index settings, mappings, and aliases.                                              | No       |
+| `priority`    | `Integer`      | The priority of the index template.                                                         | No       |
+| `composed_of` | `String array` | The names of component templates applied on a new index together with the current template. | No       |
+| `version`     | `Integer`      | Specify a version number to simplify template management. Default is `null`.                | No       |
+| `_meta `      | `Object`       | Specify meta information about the template.                                                | No       |
```

Comment:
It looks like the new table works, but all the spaces can make editing the file a bit cumbersome, especially when opened in a code editor. Would you mind changing it back to how it was before?

Reply:
I'm sorry.
Those lines were unintentionally changed by the code formatter and I'm somewhat embarrassed they slipped into the commit.
I've replaced the commit with one without the unwanted changes.

Reply:
No worries, thanks for updating it!